docs/clustering/multi-zone-setup.rst

```diff
@@ -5,124 +5,188 @@ CrateDB multi-zone setup
-file or the ``-C`` option on startup. `Read our in-depth guide
-<https://crate.io/docs/reference/configuration.html>`_ for more details on
-CrateDB configuration options.
+    Read our in-depth `configuration guide`_ for more details on CrateDB
+    configuration options.
 
-In *crate.yaml* a zone tag looks like:
+For example, you can assign a zone tag in your `configuration`_ file like this:
 
 .. code-block:: yaml
 
     node.attr.zone: us-east-1
 
-The node is tagged with a ``zone`` of the value ``us-east-1``, an availability
-zone of a cloud computing provider.
+The ``node.attr`` namespace is given a ``zone`` key and tagged with a
+``us-east-1`` value, which is an availability zone of a cloud computing
+provider.
+
+Alternatively, you can configure this at startup with a command-line option.
+For example:
+
+.. code-block:: console
+
+    sh$ bin/crate \
+            -Cnode.attr.zone=us-east-1
 
 .. NOTE::
 
-   These tags and settings can't be changed at runtime and need to be
+   These tags and settings cannot be changed at runtime and need to be
    set on startup.
 
+.. _allocation-awareness:
+
 Allocation awareness
 ====================
 
-Now tags can be set as attributes for the shard allocation awareness. We use
-the tag ``zone`` that we just assigned to the node as an attribute:
+Once you have assigned zone tags, they can be set as attributes for `shard 
+allocation awareness`_ with the 
+``cluster.routing.allocation.awareness.attributes`` setting.
+
+For example, use the ``zone`` tag that you just assigned to your node as an
+attribute in your `configuration`_ file, like this:
 
 .. code-block:: yaml
 
     cluster.routing.allocation.awareness.attributes: zone
 
-This means that CrateDB will try to allocate shards and their replicas
+This means that CrateDB will try to allocate `shards`_ and their `replicas`_
 according to the ``zone`` tags, so that a shard and its replica are not on a
 node with the same ``zone`` value.
 
-Let's add a second and a third node in a different zone (``us-west-1``) and tag
-them accordingly.
+Add a second and a third node in a different zone (``us-west-1``) and tag
+them accordingly:
 
 .. code-block:: yaml
 
     node.attr.zone: us-west-1
     cluster.routing.allocation.awareness.attributes: zone
 
-When the cluster is started, we will create a table with 6 shards and 1
-replica. The 6 shards will be distributed evenly across the nodes (2 shards on
-each node) and the replicas allocated on nodes with a different ``zone`` value
-to its primary shard.
+Now start your cluster and then `create a table`_ with 6 shards and 1 replica. 
+
+As an example, you can create such a table by executing a statement like this
+in the `CrateDB Shell`_:
+
+.. code-block:: sql
+
+    cr> create table my_table (
+          first_column integer,
+          second_column text
+        ) clustered into 6 shards
+        with (number_of_replicas = 1);
```

Comment:
SQL reserved keywords should be capitalized in example blocks and literals

https://www.postgresql.org/docs/8.1/sql-keywords-appendix.html

Reply:
Yikes! This will have to be fixed in several parts of the docs as well.